test/system/700-play.bats

```diff
@@ -278,3 +278,21 @@ status: {}
     run_podman 125 play kube - < $PODMAN_TMPDIR/test.yaml
     assert "$output" =~ "invalid annotation \"test\"=\"$RANDOMSTRING\"" "Expected to fail with annotation length greater than 63"
 }
+
+@test "podman play kube - default log driver" {
+    skip_if_journald_unavailable "system does not support journald events"
```

Comment:
No need to skip anymore as `podman` will default to k8s-file when it cannot use the journald driver. 

This way, we cover both cases.